scripts/composer/post-install.sh

```diff
@@ -1,21 +1,32 @@
 #!/bin/sh
 
+DOCUMENTROOT=web
```

Comment:
Do we want this to only be set if DOCUMENTROOT is empty, or instead be a command line option so it can be changed easily?